### DISCUSSION TITLE
make quickstart

### DISCUSSION THREAD

**Discussion by emmanuelha** [linkedin/Hoptimator#56] [Q&A]
Hello, 
I think I have a docker desktop or Kubernetes problem because when I launch a 
`sudo make quickstart`
(By the way I need to be sudo which is not great)
Then I get this:
> helm upgrade --install --atomic --set webhook.create=false flink-kubernetes-operator flink-operator-repo/flink-kubernetes-operator
> Error: Kubernetes cluster unreachable: Get "http://localhost:8080/version": dial tcp 127.0.0.1:8080: connect: connection refused
> make: *** [Makefile:30 : deploy-dev-environment] Erreur

When I run:
` kubectl get nodes`
I get this:

> NAME             STATUS   ROLES           AGE   VERSION
> docker-desktop   Ready    control-plane   13d   v1.27.2
> I am using the context docker-desktop.

Is there anyone who might have an idea ?

**Comment by emmanuelha** [comment]
I should not have used sudo now `make quickstart` works.

**Reply by ryannedolan** [reply]
I was wondering about that, but it's weird that sudo breaks things. Glad you figured it out!
